Sources/Plasma/PubUtilLib/plWinDpi/plWinDpi.h

```diff
@@ -47,7 +47,24 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include <variant>
 
 #include "hsWindows.h"
-#include <ShellScalingApi.h>
+#if __has_include(<ShellScalingApi.h>)
```

Comment:
For anyone else wondering, `__has_include` was standardized in C++17, which is our current minimum, so it's safe to use unconditionally.

So far, it seems that H'uru has preferred CMake configure-time checks instead of `__has_include`. Do we want to do that instead for consistency?

(Personally I have no issue with `__has_include` though, and I actually prefer it in most cases, because a configure-time check is more work to maintain and usually much slower.)

Reply:
It would probably be better to, for consistency's sake, do configure time checks.